workflows/prognostic_c48_run/runtime/main.py

```diff
@@ -72,11 +70,12 @@ def main():
         diag_file.flush()
 
     loop.log_global_timings()
+    return wrapper
 
 
 if __name__ == "__main__":
 
-    main()
+    wrapper = main()
```

Comment:
Feels a bit odd to return a module in this way, but not necessarily an issue. Do you know if the cleanup is still necessary?

Reply:
Locally it seems like the garbage collecting step is no longer required for the regression tests.  I pushed an update with these changes and triggered the integration tests to see if they cause any issues there.

Reply:
Okay sounds good, thanks.

Reply:
We should keep an eye on this change, maybe issues will crop up for longer runs.

Reply:
Sounds good.  I just submitted a 10-day nudged run off the latest commit in this branch, which is maybe a little bit more of a stress test.  Will merge if that completes successfully.

Reply:
The 10-day nudged run seems to have gone off without a hitch: 

```
$ podlogs nudged-fv3gfs-6603-2081873082
```

I am going to go ahead and merge.